### PR TITLE
Transform POI coords in GPX in projection 4326

### PR DESCRIPTION
**Pull request by camillemonchicourt**
Should fix - https://github.com/makinacorpus/Geotrek/issues/1351

### CODE REVIEW

File: geotrek/trekking/serializers.py [hunk 1]
```diff
@@ -29,9 +29,10 @@ class TrekGPXSerializer(GPXSerializer):
     def end_object(self, trek):
         super(TrekGPXSerializer, self).end_object(trek)
         for poi in trek.pois.all():
-            wpt = gpxpy.gpx.GPXWaypoint(latitude=poi.geom.y,
-                                        longitude=poi.geom.x,
-                                        elevation=poi.geom.z)
+            geom3d = poi.geom.transform(4326, clone=True) # GPX uses WGS84
```
Comment (leplatrem) [inline geotrek/trekking/serializers.py]: Add two spaces before ``#``

**Comment by leplatrem** [comment]
It would be useful to add some tests that make sure it actually fixes the problem :)

**Comment by leplatrem** [comment]
The fix was confirmed to be successful on the mailing-list :)

**Comment by gutard** [comment]
Superseeded by PR #1471 that adds tests and correctly handle elevation of waypoints. I close this one.

**Pull request closed without merging**